### PR TITLE
feat: Handle "forConformance: false" keys

### PR DESCRIPTION
**Pull request by Jym77**
Display accessibility requirements for documents with a `forConformance: false` key (eg, WCAG techniques).

### CODE REVIEW

File: src/components/accessibility_requirements/index.js [hunk 1]
```diff
@@ -59,17 +56,21 @@ function AccessibilityRequirementsListing({ item, listType, title, learnMore, co
...
+			conformanceTo: (
+				<>
+					<strong>Required for conformance</strong> to WAI-ARIA 1.1 specifications.
+				</>
+			),
 			baseURL: 'https://www.w3.org/TR/wai-aria-1.1/#',
 		},
 		'using-aria': {
-			conformanceTo: 'to "Using ARIA" specifications',
+			conformanceTo: 'Not required to conformance to any known conformance document.',
```
Comment (jkodu) [inline src/components/accessibility_requirements/index.js]: This message does not read right.

`Not required to confirm to any know conformance document` or `Not required for conformance ...`

File: src/components/accessibility_requirements/index.js [hunk 2]
```diff
@@ -59,17 +56,21 @@ function AccessibilityRequirementsListing({ item, listType, title, learnMore, co
 }
 
 // For documents where we provide the title, and where the item is an URL anchor
-function BasicListing({ conformanceDocument, item, mapping, listType }) {
+function BasicListing({ accessibilityDocument, item, mapping, listType }) {
 	const { conformanceTo, baseURL } = {
```
Comment (jkodu) [inline src/components/accessibility_requirements/index.js]: We can extract this to be outside the function & access it to get the required values, as it seems generic.

File: src/components/accessibility_requirements/index.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import techniquesTitles from '../../../_data/techniques-titles'
 import scUrls from '../../../_data/sc-urls'
 import React from 'react'
+import { instanceOf } from 'prop-types'
```
Comment (jkodu) [inline src/components/accessibility_requirements/index.js]: where is this used? remove it?

File: src/components/accessibility_requirements/index.js (hunk 2 above)
Comment (Jym77) [reply src/components/accessibility_requirements/index.js]: Yeah. But it's only used inside this function. Not sure…

File: src/components/accessibility_requirements/index.js (hunk 3 above)
Comment (Jym77) [reply src/components/accessibility_requirements/index.js]: Forgot to clean 😢

File: src/components/accessibility_requirements/index.js (hunk 1 above)
Comment (Jym77) [reply src/components/accessibility_requirements/index.js]: I like to keep the "known document" part as the rule could be required to conform to some internal company policy that ACT rule is not aware of…

Comment (WilcoFiers) [inline src/components/accessibility_requirements/index.js]:
```suggestion
			conformanceTo: 'Not required to conformance to any W3C accessibility recommendation.',
```

File: src/components/accessibility_requirements/index.js [hunk 4]
```diff
@@ -111,7 +112,7 @@ function TechniqueListing({ item, mapping, listType }) {
 			listType={listType}
 			title={title}
 			learnMore={title}
-			conformanceTo={`to WCAG technique ${techniqueId}`}
+			conformanceTo="Not required to conformance to any known conformance document."
```
Comment (WilcoFiers) [inline src/components/accessibility_requirements/index.js]:
```suggestion
			conformanceTo="Not required to conformance to any W3C accessibility recommendation."
```

**Pull request merged**